libs/muddle/src/rbc.cpp

```diff
@@ -534,6 +537,11 @@ void RBC::Deliver(SerialisedMessage const &msg, uint32_t sender_index)
   deliver_msg_callback_(miner_id, msg);
   lock_.lock();
 
+  if (sender_index >= parties_.size())
+  {
+    return;
+  }
+
```

Comment:
This if statement should also appear after the deliver_msg_callback_ below in line 563.

Reply:
good catch, It's in now